core/controllers/suggestion.py

```diff
@@ -174,3 +174,21 @@ def get(self):
 
         self.values.update({'suggestions': [s.to_dict() for s in suggestions]})
         self.render_json(self.values)
+
+
+class EditSuggestionHandler(base.BaseHandler):
+    """Handler for editing the submitted suggestion by reviwer before
+    accepting.
+    """
+
+    @acl_decorators.can_edit_exploration
+    def put(self, exploration_id, target_id, suggestion_id):# pylint: disable=unused-argument
```

Comment:
sir can you tell me what the main difference between the target and exploration id.
And whats the use of target id.

Reply:
@import-keshav, have you tried looking into the docstring anywhere in the file suggestion_service or suggestion_registry?

Reply:
No, and if there is written sorii for that. I will see it

Reply:
Hello @DubeySandeep in docs there is written like
``` target_id: str. The ID of the target entity being suggested to. ````
it seems like it is id of exploration. It is simillar to exploration.

Reply:
Hii @nithusha21 if i remove exploration_id i still have to use pylint pragma , since i am not using target_id in handler , it is passed just for decorating the function.